Resources/Prototypes/Body/Organs/reptilian.yml

```
@@ -1,17 +1,17 @@
-- type: entity
+﻿- type: entity
   id: OrganReptilianStomach
   parent: OrganAnimalStomach
   categories: [ HideSpawnMenu ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)